x-pack/elastic-agent/.gitignore

```diff
@@ -7,3 +7,7 @@ pkg/agent/operation/tests/scripts/configurable-1.0-darwin-x86/configurable
 pkg/agent/operation/tests/scripts/servicable-1.0-darwin-x86/configurable
 pkg/agent/transpiler/tests/exec-1.0-darwin-x86_64/exec
 pkg/agent/application/fleet.yml
+pkg/core/plugin/operation/tests/scripts/configurable/1.0/configurable
```

Comment:
What is this file about?

Reply:
hmm,  will remove. I certainly did add this by hand.